### PR TITLE
Add dmesg on pantavisor log

### PR DESCRIPTION
**Pull request by fortizc**
* Add `log.capture.dmesg` to activate or deactivate (1 or 0) this feature
* Add a fd from `/dev/kmsg` to logserver subscription

### CODE REVIEW

File: logserver.c [hunk 1]
```diff
@@ -983,6 +983,20 @@ void pv_logserver_toggle(struct pantavisor *pv, const char *rev)
 	}
 }
 
+static void logserver_capture_dmesg()
+{
+	errno = 0;
+	int fd = open("/dev/kmsg", O_RDONLY | O_NONBLOCK);
```
Comment (anibalportero) [inline logserver.c:989]: this is a good example for me to learn how this works as it is pretty simple :) Where are we closing this?

File: logserver.c [hunk 2]
```diff
@@ -983,6 +983,20 @@ void pv_logserver_toggle(struct pantavisor *pv, const char *rev)
...
+	if (fd < 0) {
+		pv_log(DEBUG, "could not susbcribe dmesg to logserver %s",
```
[the remark this answers is not in the document]
Comment (asac) [reply logserver.c:991]: yeah this is WARN if its something a setting has explicitely requested.

File: logserver.c (hunk 1 above)
Comment (asac) [reply logserver.c:989]: i am not sure if we have to close ita fter sending it to logserver ... logserver definitely closes the fd by itself so .... lets try really and be sure we use this correctly indeed.

File: logserver.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (fortizc) [reply logserver.c]: Done

File: logserver.c (hunk 1 above)
Comment (fortizc) [reply logserver.c:989]: Currently, the system relies on two ways for closing the fds, the first is the unsubscribe function that function and the second is the detection of fds in bad states. So @anibalportero is right this fd in particular never be close, maybe we will need a function to close and unsubscribe all fds when we want to shut down the system (maybe only in the pv_logserver_stop() function?)

**Pull request merged**